src/main/java/edu/kis/powp/jobs2d/features/WorkspaceFeature.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.powp.appbase.Application;
+
+import edu.kis.powp.jobs2d.drivers.SelectworkspaceMenuOptionListener;
+import edu.kis.powp.jobs2d.drivers.canva.WorkspaceCanva;
+import edu.kis.powp.jobs2d.drivers.observers.DriverLabelChangeObserver;
+
+public class WorkspaceFeature {
+    private static Application app;
+
+    public static void setupWorkspacePlugin(Application application) {
+        app = application;
+        app.addComponentMenu(WorkspaceFeature.class, "Workspaces");
+
+        DriverLabelChangeObserver driverLabelChangeObserver = new DriverLabelChangeObserver(application);
+        DriverFeature.getDriverManager().getChangePublisher().addSubscriber(driverLabelChangeObserver);
```

Comment:
Why this code is here? It is already executed in:
https://github.com/iis-powp-2025/powp_jobs2d_projects/blob/master/src/main/java/edu/kis/powp/jobs2d/features/DriverFeature.java

Reply:
Still not removed? Why?